TOC.md

```diff
@@ -64,6 +64,7 @@
     - [Monitor a TiDB Cluster](dev/how-to/monitor/monitor-a-cluster.md)
   + Migrate
     - [Overview](dev/how-to/migrate/overview.md)
+    + [Migrate from Aurora](dev/reference/tools/data-migration/from-aurora.md)
```

Comment:
Please update the link and move this line to the position before "Migrate from CSV".

```suggestion
    + [Migrate from Aurora](dev/how-to/migrate/from-aurora.md)
```

Reply:
Addressed!